src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -1012,7 +1038,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     {
         auto lock = _terminal->LockForWriting();
         _terminal->SelectAll();
-        _renderer->TriggerSelection();
+        _updateSelection(false);

```

Comment:
I'm not too sure I like this actually! I can't divine rhyme or reason as to why we would clear sometimes and not others. Is there a guiding principle you're using?

Reply:
yeah, the general intent is...
- are you using your keyboard? --> show markers
- using a mouse? --> clear markers
- search? --> clear
- toggle block selection? --> probably don't mess with it? (I'm adding this in the next commit)